src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -473,6 +512,23 @@ public void ProcessProcessName()
             Assert.Equal(_process.ProcessName, CoreRunName, StringComparer.OrdinalIgnoreCase);
         }
 
+        [Fact]
+        public void Process_SafeHandle()
+        {
+            Assert.False(_process.SafeHandle.IsInvalid);
+        }
+
+        [Fact]
+        public void Process_SessionId()
+        {
+            if (global::Interop.IsWindows)
+            {
+                uint sessionId;
+                ProcessIdToSessionId((uint)_process.Id, out sessionId);
+                Assert.Equal(sessionId, (uint)_process.SessionId);
+            }
+            else Assert.Equal(getsid(_process.Id), _process.SessionId);
+        }
```

Comment:
Nit: consider restructuring this like:
```C#
uint sessionId;
if (global::Interop.IsWindows)
{
    ProcessIdToSessionId((uint)_process.Id, out sessionId);
}
else
{
    sessionId = getsid(_process.Id);
}
Assert.Equal(sessionId, _process.SessionId);
```
That way we have the one assert, and we're simply switching by platform to get the expected value.